test/recipes/30-test_evp_data/evppkey_slh_dsa_keygen.txt

```diff
@@ -0,0 +1,850 @@
+# Copyright 2025 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+# ACVP test data for SLH-DSA keyGen generated from
+# https://github.com/usnistgov/ACVP-Server/blob/master/gen-val/json-files/SLH-DSA-keyGen-FIPS204/internalProjection.json
+# [version 53]
+
+FIPSversion = >=3.5.0
+KeyGen = SLH-DSA-SHA2-128s
+KeyName = tcId1
+Seed = AC379F047FAAB2004F3AE32350AC9A3D829FFF0AA59E956A87F3971C4D58E7100566D240CC519834322EAFBCC73C79F5
```

Comment:
These need to become
Ctrl = hexseed:
CtrlOut=hexpub:
CtrlOut = hexpriv:
Since I am removing this hardcoded rubbish.

Reply:
I saw you comment on Signal.  There is no point doing this until the SLH-DSA feature branch has been merged.  The CtrlOut won't exist until then and I don't want to pull conflicting commits in.